spec/spec_helper.rb

```diff
@@ -2,11 +2,9 @@
 
 if ENV['COVERAGE'] == 'true'
   require 'simplecov'
-  require 'coveralls'
```

Comment:
If you're removing coveralls here, you might want to remove the link from the README.

It's not like the badge ever displays correct coverage information anyway. I've reported this [multiple](https://github.com/lemurheavy/coveralls-public/issues/414) [times](https://github.com/lemurheavy/coveralls-public/issues/645) and it doesn't seem like anyone is interested in resolving it. There was some suggestion that it was github's fault, but if you go to https://coveralls.io/github/dkubb/ice_nine you can see on the right-side the coverage is reported incorrectly. Probably some kind of caching bug, but still super annoying for it to be broken for almost a year.

/rant